astro/src/content/docs/lifecycle/migrate-users/connectors/generic-connector.mdx

```diff
@@ -98,7 +98,7 @@ To use a Generic Connector:
 
 The request to your API endpoint will be delivered as JSON:
 
-<JSON title="Example Request JSON" src="login/request.json" />
```

Comment:
I think the idea behind making it the same as the login request was that it is the same object, so if we leverage `login/request.json`, it will always be up to date.

Reply:
It's actually a little different. The webhook call contains an `eventInfo` object that has the device, IP address, and user agent info, while the login call has a `metadata` object, and a `device` object inside of that, which has some fields that the metadata call doesn't (e.g. metadata.device.description).

Reply:
Hmmm. I'm confused, sorry. I don't understand what the webhook call has to do with this. Do you mean the generic API call?

My understanding is that both the generic connector and the login API use the [LoginRequest](https://github.com/FusionAuth/fusionauth-java-client/blob/master/src/main/java/io/fusionauth/domain/api/LoginRequest.java) and so should share the exact same request JSON.

What am I missing?

Reply:
Ha, sorry, wires crossed in my head. Yes, they do use the same request class, but the login api doc doesn't mention event data (event though I suppose it would accept it) and the generic connector call always includes it. So I was trying to show that this was present in the generic connector call, when it wasn't in the existing login api request json file.